.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,5 +11,5 @@ StackOverflow discussion that's relevant]
 
 - [ ] New functionality includes tests
 - [ ] All tests pass
-- [ ] RELEASE\_NOTES.md has been updated if required (not required for bugfixes, required for API changes)
+- [ ] CHANGELOG.md has been updated if required (not required for bugfixes, required for API changes)
```

Comment:
Can you remove what is in the parenthesis here? If we just have `CHANGELOG.md`, we still want a line in there for a bug fix.